.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "yarn"
+      - run: yarn
```

Comment:
Is this using Yarn 1.x? Does this need `--frozen-lockfile`?

Reply:
Well, this is library, so consumers aren't bound by the lockfile here. Leaving off the `--frozen-lockfile` will probably be closer to how this gets used in the wild.